opentaskspal/src/main/java/org/dmfs/opentaskspal/readdata/DueTime.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.opentaskspal.readdata;
+
+import android.support.annotation.NonNull;
+
+import org.dmfs.android.contentpal.Projection;
+import org.dmfs.android.contentpal.RowDataSnapshot;
+import org.dmfs.android.contentpal.projections.Composite;
+import org.dmfs.android.contentpal.projections.MultiProjection;
+import org.dmfs.iterables.elementary.Seq;
+import org.dmfs.iterators.Function;
+import org.dmfs.jems.function.BiFunction;
+import org.dmfs.optional.Optional;
+import org.dmfs.optional.adapters.FirstPresent;
+import org.dmfs.optional.composite.Zipped;
+import org.dmfs.optional.decorators.DelegatingOptional;
+import org.dmfs.optional.decorators.Mapped;
+import org.dmfs.rfc5545.DateTime;
+import org.dmfs.rfc5545.Duration;
+import org.dmfs.tasks.contract.TaskContract.Tasks;
+
+
+/**
+ * {@link Optional} for the due {@link DateTime} of a task.
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class DueTime extends DelegatingOptional<DateTime>
```

Comment:
please rename to `EffectiveDueTime` otherwise it could be mistaken for the `Due` field. This should also be mentioned in the JavaDoc.